File: sample/src/main/java/com/example/java/IfElseFlattening.java
package com.example.java;

public class IfElseFlattening {
    public void foo() {
        if (true) {
            return;
        }
        System.out.println();
        System.out.println();
    }

    public void bar() {
        if (true) {
            System.out.println();
        } else if (false) {
            System.out.println();
        } else {
            System.out.println();
        }
    }
}

File: sample/src/main/kotlin/com/example/kotlin/IfElseFlattening.kt
package com.example.kotlin

class IfElseFlattening {
    fun foo() {
        if (false) {
            return
        }
        if (false) {
            return
        }
        println()
        println()
    }

    fun bar() {
        if (true) {
            println()
        } else if (false) {
            println()
        } else {
            println()
        }
    }
}
